data/net/minecraft/resources/ResourceLocation.mapping

```diff
@@ -1,29 +1,91 @@
 CLASS net/minecraft/resources/ResourceLocation
+	COMMENT A class representing the immutable location of a resource, in terms of a path and namespace.
```

Comment:
Same bit of pedantry here, first part could be shortened down to `An immutable location of a resource,`.